src/pages/Inventory.page.js

```diff
@@ -12,4 +12,49 @@ export class InventoryPage extends BaseSwagLabPage {
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    // Generate and return random number  generateRandomNumber(0, 6)  from 0 to 5 include.
+    static generateRandomNumber(min, max){
+        return  Math.floor( Math.random()*(max - min) + min)
+    }
+
+    //parse product data by index from Catalog and return an object
+    async parseItemDataByIndexCatalog(index){
+        const productItem =  this.page.locator('.inventory_item').nth(index); 
+        return {
+            name: await productItem.locator('.inventory_item_name').innerText(),
+            description: await productItem.locator('.inventory_item_desc').innerText(),
+            price: Number((await productItem.locator('.inventory_item_price').innerText()).replace('$', '')),
+            productIndex: index,
+        }
+    }
+
+    // generate two random number (can not be equal) and return result as array 
+    async generateTwoNotEqualRandomNumber(){
+        let numberOne = InventoryPage.generateRandomNumber(0,6)
+        let numberTwo = numberOne
+        
+        while (numberOne === numberTwo) {
+            numberTwo = InventoryPage.generateRandomNumber(0, 6);
```

Comment:
You have hardcoded 6 here because of the number of items on the page, right? Lets think of moving this logic out.

Reply:
@hlazkov 
_You have hardcoded 6 here because of the number of items on the page, right?_  - **Yes**

_Lets think of moving this logic out._ - you mean change 6 ? if items will be more? I can create an array with all items and use it as parameters for this function